app/models/base_models.py

```diff
@@ -1,8 +1,12 @@
 import peewee
 from decouple import config
 
-DATABASE = peewee.PostgresqlDatabase('bothub', user='postgres',
-                                     password='postgres', host=config('BOTHUB_POSTGRES'))
+DATABASE = peewee.PostgresqlDatabase(
```

Comment:
Why aren't we use `psycopg2` - https://wiki.postgresql.org/wiki/Psycopg2_Tutorial - for that? It's the official lib to connect Python to Postgres.